code/game/machinery/suit_storage_unit.dm

```diff
@@ -586,7 +586,7 @@
 	var/electrified = 0
 
 	//Departments that the cycler can paint suits to look like.
-	var/list/departments = list("Engineering","Mining","Medical","Security","Atmos","HAZMAT","Construction","Biohazard","Emergency Medical Response","Crowd Control")
+	var/list/departments = list("Engineering","Mining","Medical","Security","Atmos","HAZMAT","Construction","Biohazard","Emergency Medical Response","Crowd Control","Sec EVA")

```

Comment:
"Sec EVA" should probably use a proper name, since any suits that can go in here are EVA suits, and "sec", while a useful abbreviation when trying to call out for help (esp when typing), is not the sort of thing you'd expect to appear on a menu selection. (I also personally object to "Atmos" for the same reason, but "Atmospherics" may well be long enough that it makes the UI clunkier)

Might it also be possible to generate this list at runtime, using initial() checks on typesof() // subtypesof() to fetch the names and whether they should be available? Ideas off the top of my head for limiting availability are either using a boolean flag, or, better yet, making station voidsuits subtypes of a base station voidsuit type, and using that as the progenitor for the list of types to pull from.

Reply:
Yeah, I just used Sec EVA at first because I wasn't sure if setting it to "Security EVA" would short out and return to "Security" settings. I need to get around to updating this part now that I've run some more tests.

I've considered something like runtime list generation, but honestly I'm not entirely sure it's worth the effort*. More importantly, it's a bit above my level of expertise; this whole thing is more or a less an amateur fiddling at things and being pleasantly surprised when an idea they had actually worked.

*Although one could reasonably argue that whoever made this thing in the first place clearly didn't put in much effort themselves if they didn't think to use initial()...

Reply:
Also, with a little hindsight: the main thing I wanted to avoid with the Security EVA suits is using edgy buzzwords like "Tactical" or "Elite".

That job is reserved for the ERT and Security attracts enough easily-inflated egos already. Problem is "Riot" is basically synonymous with "Crowd Control", and it doesn't make sense to call them "Surplus" (as the comments imply they are) when they're *better* than standard-issue suits (unless the intent is to mean 'military surplus', but in that case there are better terms I think).